### DISCUSSION TITLE
Add consumed kWh at the charger

### DISCUSSION THREAD

**Discussion by riesi77** [teslamate-org/teslamate#811] [Ideas]
**What would you like to be added**:
I'd like to have a field - ideally with UI - to add the amount of kWh that have been taken from the charger. In my recent charge for instance I got 38 kWh logged that went into the car, but the Charger stated 39.1.

**Why is this needed**:
This would allow statistics about the efficiency of charging and the energy that gets lost during charging.
It would also allow precise calculation of charging costs (would be great to also have a ui to enter the costs per kwh depending on the location of the charger)

**Additional context**:

**Comment by petariga** [comment]
Duplicate request to #684
